src/main/plugins/org.dita.html5/sass/domains/_hazard.scss

```diff
@@ -22,3 +22,20 @@
 
 .typeofhazard {
 }
+
+.hazardstatement--caution {
```

Comment:
Since this is SCSS, my preference would be use

```scss
.hazardstatement {
  &--caution {
    background-color: #FFD100;
  }
  &--danger {
    background-color: #C8102E;
  }
...
}
```